src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day calendar--month-length_31">
```

Comment:
Modifier of month length not work correctly.
Look on the screen
With modifier calendar--month-length_29 should be visible only 29 days in calendar
![Screenshot from 2023-05-03 12-36-15](https://user-images.githubusercontent.com/33029864/235882223-a37c474c-2073-43ce-b7b2-ff71509b86ef.png)